update.py

```diff
@@ -22,41 +23,45 @@
 version, all_versions = all_versions[0], all_versions[1:]
 
 last = version.split('.')
-keep_version = [version]
+keep_version = ['snapshot']
+
+keep_version.append(version)
 
 for cur in all_versions:
     if cur < MIN_VERSION:
         break
 
     tmp = cur.split('.')
-## Check Minor
+    # Check Minor
     if len(tmp) >= 2 and tmp[1].isdigit() and tmp[1] < last[1]:
         keep_version.append(cur)
         last = tmp
-## Check Major
+    # Check Major
     elif len(tmp) > 1 and tmp[0].isdigit() and tmp[0] < last[0]:
         keep_version.append(cur)
         last = tmp
 
 for version in keep_version:
     for variant in VARIANTS:
-        if variant == 'ubuntu':
-            dockerfile = 'docker-images/%s/Dockerfile' % version[0:3]
-            travis.append(' - VERSION=%s' % version[0:3])
+        if version == 'snapshot':
```

Comment:
keep ubuntu in a dedicated folder so it's consistent. I"ll have to update docker hub though